src/client/components/style/labels.js

```diff
@@ -55,12 +55,9 @@ export function stylePropsToLabelPos(h, v) {
 
 
 export function PositionButton({ value, selected, onClick }) {
-  const style = { cursor:'pointer', display:'flex', alignItems:'center', justifyContent:'center' };
-  if(selected)
-    style.color = '#757ce8';
   return (
-    <div onClick={() => onClick(value)} style={style}>
-      <b>{value.label}</b>
+    <div className={'label-position-widget-button ' +  'label-position-widget-button-' + value.label + ' '  + (selected ? 'label-position-widget-selected-true' : 'label-position-widget-selected-false')} onClick={() => onClick(value)}>
```

Comment:
This classname expression is quite long.  Maybe this can be simplified with template strings.

Reply:
Yeah.  We should probably be using something like that `createClassName` utility, if we're not already.  I think we used that in Biofactoid

Reply:
Amended